qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_presenter.py

```diff
@@ -162,6 +164,9 @@ def update_selected_parameter_combination_workspace(self) -> None:
             self.view.current_dataset_name = dataset_name
             # update the x range for the fit
             start_x_list, end_x_list = self.model._get_new_start_xs_and_end_xs_using_existing_datasets([dataset_name])
+            # update values in context
+            self.model.set_current_start_and_end_x(start_x_list[0], end_x_list[0])
```

Comment:
If I understand correctly, if start > end then it will not update the model but it will still update the view? Is that intentional?

Reply:
I think the view checks and if its the wrong way around then swaps it (if it is). That triggers this again as the values in the view change.